client/src/App.js

```diff
@@ -692,17 +693,7 @@ class App extends Component {
         )
       default:
         return (
-          <Route exact path={`/`} render={(props) => 
```

Comment:
What this causing the fleeting error on login earlier?

Reply:
If role was being set to null, then yes! The purpose of this was to make it a little smoother when logging in (it would flash a placeholder screen before it properly fetched the profile data) - but this meant if you logged off from anywhere that wasn't news, it would say page not found (/profile doesn't exist in the default route case)
Because of changes to the initial data fetch, and other changes to make the current header possible, the default route didn't need to exist and could just redirect to /login